py_out/quick-start/Qiskit Standalone-Quantum Volume-sampler.py

```diff
@@ -0,0 +1,13 @@
+from qiskit.primitives import Sampler
+sampler = Sampler()
+
+# Build circuit
+from qiskit.circuit.library import QuantumVolume
+circuit = QuantumVolume(5)
+circuit.measure_all()
+
+# Run the circuit and plot result distribution
+from qiskit.visualization import plot_histogram
+job = sampler.run(circuit)
+quasi_dist = job.result().quasi_dists[0]
+plot_histogram(quasi_dist)
```

Comment:
I think the other examples go directly with `print`

Reply:
Anyway we decided to use QV for transpiling without running, so we don't need to print nor plot the results.